src/data/nat/digits.lean

```diff
@@ -148,6 +149,30 @@ begin
   { dsimp [of_digits], rw ih, },
 end
 
+private lemma of_digits_eq_sum_map_with_index_aux (b : ℕ) (l : list ℕ) :
```

Comment:
We tend not to use the private modifier in mathlib.

Reply:
Removed.